addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -1028,9 +1028,13 @@ const SelectUserValueWidget = BaseSelectionUserValueWidget.extend({
             textContent = "/";
         }
 
-        this.menuTogglerEl.textContent = textContent;
-        if (this.menuTogglerItemEl) {
+        if (!oldTogglerEl && !this.menuTogglerItemEl) {
+            this.menuTogglerEl.textContent = textContent;
+        } else if (!oldTogglerEl) {
+            this.menuTogglerEl.textContent = textContent;
             this.menuTogglerEl.appendChild(this.menuTogglerItemEl);
+        } else if (!oldTogglerEl.isEqualNode(this.menuTogglerItemEl)) {
+            oldTogglerEl.replaceWith(this.menuTogglerItemEl);
```

Comment:
No, because then the contents of `menuTogglerEl` get replaced. The point of the fix is avoiding a replacement whenever the new contents are equal to the old contents.